src/web/rustdoc.rs

```diff
@@ -262,7 +262,9 @@ pub fn rustdoc_html_server_handler(req: &mut Request) -> IronResult<Response> {
 
     content.full = file_content;
     let crate_details = cexpect!(CrateDetails::new(&conn, &name, &version));
-    let (path, latest_version) = if let Some(latest_version) = latest_version(&crate_details.versions(), &version) {
+
+    let (path, latest_version) = if !crate_details.is_latest_version() {
+        let latest_version = crate_details.latest_version().to_string();
```

Comment:
This now calculates the latest version twice. Not a giant deal since it's now just a field, but it would be nice to only calculate once.